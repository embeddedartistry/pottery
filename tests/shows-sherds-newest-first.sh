pottery init
TEST_TIME=1992-01-12T16:34:23Z pottery note "First note"
TEST_TIME=1992-01-12T17:00:00Z pottery note "Second note."
TEST_TIME=1992-01-13T16:34:23Z pottery note "It's a new day."
pottery show
