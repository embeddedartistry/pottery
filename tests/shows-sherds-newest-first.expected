pottery init
TEST_TIME=1992-01-12T16:34:23Z pottery note "First note"
docs/project-history/1992/1992-01/1992-01-12T16:34:23Z_note_FAKEUNIQUEID.md
TEST_TIME=1992-01-12T17:00:00Z pottery note "Second note."
docs/project-history/1992/1992-01/1992-01-12T17:00:00Z_note_FAKEUNIQUEID.md
TEST_TIME=1992-01-13T16:34:23Z pottery note "It's a new day."
docs/project-history/1992/1992-01/1992-01-13T16:34:23Z_note_FAKEUNIQUEID.md
pottery show
1992-01-13T16:34:23Z

It's a new day.

1992-01-12T17:00:00Z

Second note.

1992-01-12T16:34:23Z

First note

